control.py

```diff
@@ -17,69 +22,215 @@ def hello():
+        return jsonify(locacao_from_db)
+    elif request.method == "POST":
+        data = locacoes_from_web(**request.json)
+        if valida_locacoes(**data):
+            id_locacoes = insert_locacoes(**data)
+            data_add = get_locacoes(id_locacoes)
+            return jsonify(locacoes_from_db(data_add))
+        else:
+            return jsonify({"Erro": "Valor nao adicionado"})
+
+
+@app.route("/locacoes/<int:id>", methods=["DELETE", "PUT", "PATCH"])
+def locacoes_delete_update(id):
+    if request.method == "DELETE":
+        try:
+            deletar_locacoes(id)
+            return jsonify({"Valor": "Deletado"})
+        except:
+            return jsonify({"Erro": "Valor nao Excluido"})
+    elif request.method == "PUT" or "PATCH":
+        locacao = locacoes_from_web(**request.json)
+        if valida_locacoes(**locacao):
+            update_locacoes(id, **locacao)
+            locacao_nova = get_locacoes(id)
+            return jsonify(locacoes_from_db(locacao_nova))
+        else:
+            return jsonify({"Erro": "Valor nao foi alterado"})
+
+
+@app.route("/filmes", methods=["GET", "POST"])
+def filmes_getpost():
+    if request.method == "GET":
+        titulo = filmes_titulos_from_web(**request.args)
+        filmes = select_filme(**titulo)
+        filme_from_db = [filmes_from_db(filme) for filme in filmes]
+        return jsonify(filme_from_db)
+    elif request.method == "POST":
+        filme = filmes_from_web(**request.json)
+        if valida_filme(**filme):
+            id_filme = insert_filme(**filme)
+            filme_add = get_filme(id_filme)
+            return jsonify(filmes_from_db(filme_add))
+        else:
+            return jsonify({"Erro": "Valor nao adicionado"})
 
 
-@app.route("/usuarios", methods=["POST"])
-def usuarios_post():
-    usuario = usuario_from_web(**request.json)
-    if valida_usuario(**usuario):
-        id_usuario = insert_usuario(**usuario)
-        usuario_cadastrado = get_usuario(id_usuario)
-        return jsonify(usuario_from_db(usuario_cadastrado))
-    else:
-        return jsonify({"erro": "Usuário inválido"})
+@app.route("/filmes/<int:id>", methods=["DELETE", "PUT", "PATCH"])
+def filmes_delete_update(id):
+    if request.method == "DELETE":
+        try:
+            deletar_filme(id)
+            return jsonify({"Valor": "Deletado"})
+        except:
+            return jsonify({"Erro": "Valor nao Excluido"})
+    elif request.method == "PUT" or "PATCH":
+        titulo = filmes_from_web(**request.json)
+        if valida_filme(**titulo):
+            update_filme(id, **titulo)
+            filme_novo = get_filme(id)
+            return jsonify(filmes_from_db(filme_novo))
+        else:
+            return jsonify({"Erro": "Valor nao foi alterado"})
 
 
-@app.route("/generos", methods=["GET", "POST", "DELETE", "PATCH"])
-def genero():
+@app.route("/pagamento", methods=["GET", "POST"])
+def filmes_getpost():
```

Comment:
```suggestion
def filmes_get_post():
```
Importante separar as palavras com _, e manter consistência na nomenclatura